spec/Powertrain/Battery.vspec

```diff
@@ -196,6 +196,12 @@ Range:
   unit: m
   description: Remaining range in meters using only battery.
 
+Time:
```

Comment:
I like the new signals - but should they possibly be called `TimeRemaining` or similar? If we just call them `Time` it might be difficult to know what they are intended for if just looking at the name.

Reply:
Yes, we can call it TimeRemaining.
I wanted to apply the same logic as for the "range".

Range:
  datatype: uint32
  type: sensor
  unit: m
  description: Remaining range in meters using only battery.

But then, shouldn't Range be RangeRemaining or RemainingRange ? Because Range could be the maximum distance you can travel with a full battery and RemainingRange the distance you can travel with the current charge level. I say this in order to keep consistency, this would be a change to be made in another branch of course.

What do you think about this?